integration-tests/src/test/java/oracle/kubernetes/operator/ItManagedCoherence.java

```diff
@@ -238,6 +259,13 @@ private void createDomainAndDeployApp(Map<String, Object> domainMap, String doma
       domainUid + "-new-managed-server2",
     };
     verifyServersStatus(domain, pods, domainUid);
+    Thread.sleep(60 * 1000);
```

Comment:
does this need to be in a try/catch @rjeberhard ?

Reply:
There must be a try/catch at a higher level... yes, the method signature throws Exception.  This is okay if on InterruptedException it's correct that the test should fail.

Reply:
Is the sleep really needed here? If not remove or reduce? we usually don't put hard sleep's like this in test, instead wait for a condition in a loop with small sleep time.

Reply:
Will remove the sleep. Will not add try/catch. reason - what Ryan said.